.github/workflows/test-dev.yml

```
@@ -79,6 +177,9 @@ jobs:
     runs-on: ubuntu-24.04
     timeout-minutes: 10
     needs: poetry-install
+    env:
+      IMAGE_TAG: ${{ needs.devcontainer.outputs.IMAGE_TAG }}
```

actionlint (1.7.4)

property "devcontainer" is not defined in object type {poetry-install: {outputs: {}; result: string}}

(expression)